sig-cli/annual-report-2021.md

```diff
@@ -0,0 +1,71 @@
+# SIG CLI Annual Report 2021
+
+This report reflects back on CY 2020 and was written in March 2021.
+
+## Operational
+
+* How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+
+    * Is your README accurate? have a CONTRIBUTING.md file?
+        * Yes, [README.md](https://github.com/kubernetes/community/blob/master/sig-cli/README.md) is accurate and up-to-date.
+        * Our [CONTRIBUTING.md](https://github.com/kubernetes/community/blob/master/sig-cli/CONTRIBUTING.md) requires an update as a follow-up to our recent completion of the [move to staging](https://github.com/kubernetes/enhancements/issues/1020).
+
+    * All subprojects correctly mapped and listed in sigs.yaml?
+        * The current list is up-to-date, but this will soon require an update as soon as [KUI moves to kubernetes-sig](https://github.com/kubernetes/org/issues/2461).
+
+    * What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+        * We have two recurring meetings:
+        1. Our main meeting happens biweekly, during those calls we always reserve time for important announcements, such as important release dates, recognitions, etc. After that we leave time for newcomers to introduce themselves. The remaining part of the call is devoted to discuss items which are in the agenda. We close the call with a round of standups from subprojects.
+        2. Bug scrub every four weeks, during which we go through issues and pull requests, assigning them to contributors who are interested in working on them.
+        All of the meetings are recorded and available online, all the meeting invites are up-to-date and present in community calendar.
+
+* How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+    * As mentioned above, all the subprojects have a spot to report their progress during standups at the end of our biweekly calls. OWNERS files are up-to-date.
```

Comment:
is there a process by which you look to see if they are up to date?

Reply:
There's no particular process, we periodically (~once a year) review those. Each area is responsible for their own OWNERS file.